### PR TITLE
fix blocking in dumps

### PR DESCRIPTION
**Pull request by MarinPostma**
This PR fixes two blocking calls in the dump process.

### CODE REVIEW

File: meilisearch-lib/src/dump/handler.rs [hunk 1]
```diff
@@ -78,15 +79,22 @@ mod real {
...
             create_dir_all(&temp_dump_path.join("indexes")).await?;
 
-            // TODO: this is blocking!!
-            AuthController::dump(&self.db_path, &temp_dump_path)?;
+            let db_path = self.db_path.clone();
+            let temp_dump_path_clone = temp_dump_path.clone();
+            tokio::task::spawn_blocking(move || -> Result<()> {
+                AuthController::dump(db_path, temp_dump_path_clone)?;
+                Ok(())
+            })
```
[the remark this answers is not in the document]
Comment (ManyTheFish) [reply meilisearch-lib/src/dump/handler.rs:96]: ok. 👍

Comment (irevoire) [inline meilisearch-lib/src/dump/handler.rs:96]:
```suggestion
            tokio::task::spawn_blocking(move || AuthController::dump(db_path, temp_dump_path_clone))
```

That doesn't work?

**Review by irevoire — APPROVED** [review]

**Review by ManyTheFish — APPROVED** [review]
Thanks Marin le mops!

Bors merge

**Review by irevoire — APPROVED** [review]
Did bors just ignored you?

bors merge

**Pull request merged**